pkgs/applications/networking/instant-messengers/ring-daemon/restbed.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, fetchFromGitHub
+, cmake
+, asio
+, openssl
+, patches
+}:
+
+stdenv.mkDerivation rec {
+  name = "restbed-${version}";
+  version = "34187502642144ab9f749ab40f5cdbd8cb17a54a";
```

Comment:
I guess this is strictly an internal component, but in general using revisions for version strings is not recommended. It is important that `new > old` so that name based package management will be able to pick up on updated versions. You can use the date of the revision in `year-month-day` format instead.